reference/checkouts.v3.yml

```diff
@@ -88,6 +88,49 @@ paths:
         AllowDynamicQueryParameters: false
         AllowDynamicFormParameters: false
         IsMultiContentStreaming: false
+    put:
+      description: |-
+        Change customer message to an existing *Checkout*.
+      tags:
+        - Checkout
+      operationId: CheckoutsByCheckoutIdPut
+      produces:
+        - application/json
+      parameters:
+        - name: checkoutId
+          in: path
+          type: string
+          required: true
+          description: Id of the Checkout
```

Comment:
```suggestion
          description: ID of the Checkout
```

Reply:
changed this one to recommended. thanks!